skycoin/skywire
# Replaced go:generate with go:embed

Did you run `make format && make check`?

Fixes #737 

 Changes:	
- replaced //go:generate with //go:embed in skychat

How to test this PR:
1. Set up [Interactive test environments](https://github.com/SkycoinPro/skywire-services/blob/master/docs/InteractiveEnvironments.md).
2. `git checkout` this PR branch.
3. Follow the instructions to Test with local changes from [Interactive test environments](https://github.com/SkycoinPro/skywire-services/blob/master/docs/InteractiveEnvironments.md).
4. Run `make integration-run-generic`
5. Go to `http://localhost:8001/` and `http://localhost:8002/` and check if its working as intended.
